client/.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "plugin:react/recommended",
+        "airbnb",
+        "prettier"
+    ],
+    "overrides": [
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+        "semi": ["error", "always"],
+        "quotes": ["error", "double"]
```

Comment:
When I run the `npm start` I get:

```
Failed to compile.

[eslint]
src/App.js
  Line 1:8:   Strings must use doublequote                   quotes
  Line 2:21:  Strings must use doublequote                   quotes
  Line 3:20:  Strings must use doublequote                   quotes
  Line 4:24:  Strings must use doublequote                   quotes
  Line 8:5:   'React' must be in scope when using JSX        react/react-in-jsx-scope
  Line 8:5:   JSX not allowed in files with extension '.js'  react/jsx-filename-extension
  Line 9:7:   'React' must be in scope when using JSX        react/react-in-jsx-scope
  Line 10:7:  'React' must be in scope when using JSX        react/react-in-jsx-scope
  Line 11:7:  'React' must be in scope when using JSX        react/react-in-jsx-scope

src/components/content/Content.jsx
  Line 2:10:  'React' must be in scope when using JSX  react/react-in-jsx-scope

src/components/footer/Footer.jsx
  Line 2:10:  'React' must be in scope when using JSX  react/react-in-jsx-scope

src/components/navigation/Navigation.jsx
  Line 2:10:  'React' must be in scope when using JSX  react/react-in-jsx-scope

src/index.js
  Line 1:19:  Strings must use doublequote                   quotes
  Line 2:22:  Strings must use doublequote                   quotes
  Line 3:8:   Strings must use doublequote                   quotes
  Line 4:17:  Strings must use doublequote                   quotes
  Line 6:58:  Strings must use doublequote                   quotes
  Line 8:3:   JSX not allowed in files with extension '.js'  react/jsx-filename-extension

Search for the keywords to learn more about each error.
ERROR in [eslint]
src/App.js
  Line 1:8:   Strings must use doublequote                   quotes
  Line 2:21:  Strings must use doublequote                   quotes
  Line 3:20:  Strings must use doublequote                   quotes
  Line 4:24:  Strings must use doublequote                   quotes
  Line 8:5:   'React' must be in scope when using JSX        react/react-in-jsx-scope
  Line 8:5:   JSX not allowed in files with extension '.js'  react/jsx-filename-extension
  Line 9:7:   'React' must be in scope when using JSX        react/react-in-jsx-scope
  Line 10:7:  'React' must be in scope when using JSX        react/react-in-jsx-scope
  Line 11:7:  'React' must be in scope when using JSX        react/react-in-jsx-scope

src/components/content/Content.jsx
  Line 2:10:  'React' must be in scope when using JSX  react/react-in-jsx-scope

src/components/footer/Footer.jsx
  Line 2:10:  'React' must be in scope when using JSX  react/react-in-jsx-scope

src/components/navigation/Navigation.jsx
  Line 2:10:  'React' must be in scope when using JSX  react/react-in-jsx-scope

src/index.js
  Line 1:19:  Strings must use doublequote                   quotes
  Line 2:22:  Strings must use doublequote                   quotes
  Line 3:8:   Strings must use doublequote                   quotes
  Line 4:17:  Strings must use doublequote                   quotes
  Line 6:58:  Strings must use doublequote                   quotes
  Line 8:3:   JSX not allowed in files with extension '.js'  react/jsx-filename-extension

Search for the keywords to learn more about each error.

webpack compiled with 1 error

```

There are other rules but `"quotes": ["error", "double"]` is the oposite of `"singleQuote": true` from prettier. We should same rules for prettier and eslint.

Would be nice to have all issues regarding eslint to be fixed in this PR.
Also, I don't see instructions for other teammates on what extensions to install in VSCode for eslint and prettier, and how to configure the editor